sdk/eventhub/azure-eventhub/azure/eventhub/_producer.py

```diff
@@ -185,41 +277,29 @@ def _on_outcome(self, outcome, condition):
 
     def _wrap_eventdata(
         self,
-        event_data,  # type: Union[EventData, EventDataBatch, Iterable[EventData]]
+        event_data_batch,  # type: EventDataBatch
         span,  # type: Optional[AbstractSpan]
         partition_key,  # type: Optional[AnyStr]
     ):
-        # type: (...) -> Union[EventData, EventDataBatch]
-        if isinstance(event_data, EventData):
```

Comment:
Couldn't see where the processing of a single EventData got moved to?

Reply:
Actually I guess you can only send by batch at the moment?

Reply:
correct, currently we only accept the batch/list object for the public send API.
so this is a simplification for the internal logic.